job-dsls/jobs/tools/tools_slave_image_build.groovy

```diff
@@ -13,7 +17,8 @@ cd jenkins-slaves
 rsync -av bxms-jenkins/jenkins-image-extra-bits/rhba-osbs/ansible/ ansible
 rsync -av bxms-jenkins/jenkins-image-extra-bits/rhba-sourceclear-integration/ansible/ ansible
 
-wget --no-check-certificate https://rhba-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/userContent/packer
+wget $packerUrl -O packer.zip
```

Comment:
is `--no-check-certificate` not needed anymore?

Reply:
@winklerm @Ginxo  it seems not

Reply:
Should not be needed since packer is published on a server with a certificate signed by a public CA.